libr/bin/format/pyc/marshal.c

```diff
@@ -1046,10 +1046,16 @@ static pyc_object *get_object(RBuffer *buffer) {
 		break;
 	case TYPE_CODE_v0:
 		ret = get_code_object (buffer);
+		if (!ret) {
+			return NULL;
+		}
 		ret->type = TYPE_CODE_v0;
 		break;
 	case TYPE_CODE_v1:
 		ret = get_code_object (buffer);
+		if (!ret) {
```

Comment:
What about if (ret) so it will return something, even partial at least?

Reply:
the issue is on the null deref. i need to have a return NULL anyway. if you do not like this, i can use r2 macros.